scraper/buildkite.yaml

```diff
@@ -1,15 +1,14 @@
 env:
   DATA_DIR: /mnt/storage-1/solana-snapshot-manager
-  PNPM_HOME: "$$DATA_DIR/pnpm"
-  TAG: e87603e
-  NODE_OPTIONS: --max-old-space-size=16384
+  SNAPSHOT_URL: http://api.mainnet-beta.solana.com/snapshot.tar.bz2
 
 agents:
   queue: "snapshots"
 
 steps:
-  - command: echo "--> Concurrency gate"
-    concurrency_group: 'solana-snapshot-manager/parsing'
+  - label: ":closed_lock_with_key: Concurrency gate lock"
+    command: echo "--> Start of concurrency gate"
+    concurrency_group: 'solana-snapshot-manager/parsing2'
```

Comment:
this should be in the same concurrency group as the other parser now as the usage of resources will be similar and both cannot run at the same time